week-1/K-functions-parameters/exercise2.js

```diff
@@ -1,9 +1,11 @@
 // Declare your function first
-
+function divide() {
+  return 3 / 4;
```

Comment:
Here, we want to use the values that are passed into the function like in the other exercise. So you need to modify it from `function divide() {` to `function divide(num1, num2) {` and change the return statement too.